Source/ThirdParty/libwebrtc/Source/third_party/libwebm/webm_parser/src/webm_parser.cc

```diff
@@ -75,7 +75,7 @@ class WebmParser::DocumentParser {
             return status;
           }
           child_metadata_.id = id_parser_.id();
-          child_metadata_.size = size_parser_.size();
+          child_metadata_.size = child_metadata_.id == Id::kSegment ? kUnknownElementSize : size_parser_.size();
```

Comment:
We probably need to keep track of changes like this since we might not upstream the changes.
In libwebrtc, all webkit specific changes are marked with `#if defined(WEBRTC_WEBKIT_BUILD)`

Reply:
new test would fail if it happened. I will add a similar macro to flag a webkit only change

Reply:
done